src/app/globals.css

```diff
@@ -170,8 +179,41 @@ a {
 
 @font-face {
   font-family: "Poppins";
-  src: url("../../public/fonts/Poppins/Poppins-BlackItalic.ttf") format("truetype");
+  src: url("../../public/fonts/Poppins/Poppins-BlackItalic.ttf")
+    format("truetype");
   font-weight: 900;
   font-style: italic;
   font-display: swap;
 }
+.testimonials-swiper .swiper {
+  width: 100%;
+  padding: 50px 80px !important;
+}
+
+.testimonials-swiper .swiper-slide:not(.swiper-slide-active):not(.swiper-slide-prev):not(
+    .swiper-slide-next
+  ) {
+  visibility: hidden !important;
+  opacity: 0 !important;
+  transition: all 0.3s ease !important;
+}
+
+.testimonials-swiper .swiper-slide {
+  transition: all 0.3s ease !important;
+}
+
+.testimonials-swiper .swiper-slide-active,
+.testimonials-swiper .swiper-slide-prev,
+.testimonials-swiper .swiper-slide-next {
+  visibility: visible !important;
+  opacity: 1 !important;
+}
+
+.testimonials-swiper .swiper-slide-prev,
+.testimonials-swiper .swiper-slide-next {
+  opacity: 0.8 !important;
+}
+
+.testimonials-swiper .swiper-pagination-bullet {
+  background: #707070 !important;
+}
```

Comment:
Please move the entirety of these styles to a wrapper Box above the `<Swiper >` component and place these styles in the `sx={{ }}`.

We try to not use css unless **absolutely** necessary, and i donot see how this is necessary

Reply:
Please see that if all this css can't be included in the `style` prop of the `<Swiper >` component first and foremost, if not, then and only then add as nested styled into the `TestimonialsWrapper`